frontend/wallet/src/Page.re

```diff
@@ -20,11 +20,33 @@ let make = (~message) => {
       Js.log2("Failed to parse route: ", url.hash);
       (None, `Error(`Json_parse_error));
     | Some({Route.path, settingsOrError}) =>
+      let closeModal = () => Router.navigate({path: Home, settingsOrError});
       Js.log3("Got path, settingsOrError: ", path, settingsOrError);
       (
         switch (path) {
-        | Route.Path.Send => Some(<Send />)
-        | DeleteWallet => Some(<Delete />)
+        | Route.Path.Send =>
+          Some(
+            <Send
+              closeModal
+              myWallets=[
+                {
+                  Wallet.key: PublicKey.ofStringExn("BK123123123"),
+                  balance: 100,
+                },
+                {
+                  Wallet.key: PublicKey.ofStringExn("BK8888888"),
+                  balance: 783,
+                },
+              ]
+              settings={
+                switch (settingsOrError) {
+                | `Settings(settings) => settings
+                | _ => failwith("Bad; we need settings")
```

Comment:
when we have a proper no-settings failure state we'll have already pattern-matched out settings by now